app/server/lib/ExportTableSchema.ts

```diff
@@ -0,0 +1,122 @@
+import * as express from 'express';
+import {ActiveDoc} from 'app/server/lib/ActiveDoc';
+import {DownloadOptions, exportTable} from 'app/server/lib/Export';
+import { ApiError } from 'app/common/ApiError';
+
+interface ExportColumn {
+  id: number;
+  colId: string;
+  label: string;
+  type: string;
+  widgetOptions: any;
+  description?: string;
+  parentPos: number;
+}
+
+export async function downloadTableSchema(
```

Comment:
The name feels a bit misleading, since it isn't downloading at this time. Maybe `collectTableSchemaInFrictionlessFormat`? A bit of a mouthful I know but we could easily end up with endpoints describing the schema in multiple formats. Could you add a docstring with a link to the spec you are following? And a return type?